src/Sql/dbo/Stored Procedures/Organization_DeleteById.sql

```diff
@@ -37,11 +37,24 @@ BEGIN
     WHERE
         [OrganizationId] = @Id
 
+    DELETE cu
```

Comment:
i think we use upper case for these abbreviations

Reply:
changed